abstraction/src/lib.rs

```diff
@@ -40,12 +46,20 @@ impl<S: Monoid> Iterator for PowerIterator<S> {
 /// - the binary operation is associative. a(bc) = (ab)c for all a, b, c in S
 ///   This isn't easily representable in a type, so it's up to the
 ///   implementation to make sure this is valid.
```

Comment:
Update this description to describe the monoid in terms of all its properties for less cross-referencing